website/i18n/en.json

```diff
@@ -214,7 +214,15 @@
       },
       "development/extensions-core/kafka-ingestion": {
         "title": "Apache Kafka ingestion",
-        "sidebar_label": "Apache Kafka"
```

Comment:
I don't think we need to include en.json, but not 100% sure. @sthetland do you know?